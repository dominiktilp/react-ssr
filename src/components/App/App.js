import React, { Component } from 'react';

import Header from '../Header/Header';
import Detail from '../Detail/Detail';

class App extends Component {

  render() {
    return (
      <div>
        <Header />
        <h1>This is SSR App.</h1>
        <Detail initState={this.props.initState} />
      </div>
    );
  }

}

App.fetchData = function appFetchData() {
  return Detail.fetchData();
};

export default App;
