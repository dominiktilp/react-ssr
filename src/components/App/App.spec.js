import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

test('App will render h1', () => {
  const component = renderer.create(
    <App />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
