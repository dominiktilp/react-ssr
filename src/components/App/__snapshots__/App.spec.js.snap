// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App will render h1 1`] = `
<h1>
  This is SSR App
</h1>
`;
